extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -215,6 +219,25 @@ export class SqlDatabaseTree {
 		return searchContainer;
 	}
 
+	@debounce(500)
+	private _filterTableList(value: string): void {
+		if (this._databaseTableValues && value && value.length > 0) {
+			const filter: number[] = [];
+			this._databaseTableValues.forEach((row, index) => {
+				const cell: any = row[1]?.value;
```

Comment:
I believe this should be DeclarativeTableCellValue. Try to avoid using any whenever you can so we can take advantage of the type-safety that Typescript brings.